dev-environment.yml

```diff
@@ -0,0 +1,17 @@
+name: mosaic-dev
+channels: 
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python >= 3.9
+  - cartopy
+  - cmocean
+  - h5netcdf
+  - hdf5
+  - netcdf4
+  - matplotlib
+  - numpy
+  - pyproj
+  - requests
+  - scipy
```

Comment:
I would remove these since you're not explicitly using them.

Reply:
I'll drop `requests` since we don't use it yet. I added `scipy` as part of the [optional dependencies](https://docs.xarray.dev/en/stable/getting-started-guide/installing.html#for-netcdf-and-io) to xarray, in order to support netCDF i/o.

Reply:
Okay, got it.